docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.4
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
```

Comment:
This line was an attempt to allow reaching `calypso.localhost` from another machine... Maybe it's just me, but I can't get it to work (I can still reach it by the host's ip address though, which is strange). I can reach other services I start in docker by hostname, but not calypso. There might be some deeper magic at play, or it is something dumb that I'm overlooking. Let me know if you try it and it works.